packages/matchbox/src/components/Radio/Group.js

```diff
@@ -16,7 +16,7 @@ const StyledGroup = styled('fieldset')`
 `;
 
 function Group(props) {
-  const { children, label, labelHidden, required, optional, ...rest } = props;
+  const { children, label, labelHidden, optional, ...rest } = props;
```

Comment:
Should probably have a `ScreenReaderOnly` with required or set aria-required="true" on the input